src/matUtils/README.md

```diff
@@ -9,59 +9,71 @@ matUtils is a toolkit for working with mutation annotated tree protobuf files, t
 
-**-f**: Set the minimum allele frequency in samples to find the best clade root node (default = 0.9)
+**-e**: Select samples to extract by whether they have less than the indicated number of equally parsimonious placements. Note: adds significantly to runtime with large sample selections.
 
-## Convert 
+**-a**: Select samples to extract by whether their parsimony score (terminal branch length) is less than the indicated value.
 
-The convert subcommand yields non-protobuf format files representing the tree. This can be a newick format file and/or a VCF representing each sample's variations from the reference. 
+**-r**: Automatically select two samples per unique clade identifier to extract.
 
-### Specific Options
+**-p**: Prune the samples indicated by other arguments instead of extracting them to create the subtree.
 
-**-v**: Output VCF file 
+**-d**: Set the directory to save output files. Directory must already exist. Default is current directory
 
-**-t**: Output Newick tree file
+**--sample-paths**: Write the path of mutations defining each selected sample to a file.
 
-**-n**: Do not include sample genotype columns in VCF output. Used only with -v
+**--clade-paths**: Write the path of mutations defining each clade in the subtree after sample extraction to a file.
 
-## Mask
+**-v**: Convert the subtree to a VCF and write it to the indicated file.
 
-The mask subcommand restricts indicated sample names and returns a masked protobuf.
+**-n**: Do not include sample genotype columns in the VCF output. Used only with -v
 
-### Specific Options
+**-o**: Create a new MAT .pb file representing the extracted subtree. 
 
-**-o**: Output mutation-annotated tree file (REQUIRED)
+**--collapse-tree**: Collapse the subtree before saving.
 
-**-s**: Use to mask specific samples from the tree. 
+**-t**: Write a newick string representing the extracted subtree to the indicated file.
 
-## Describe
+## Summary
 
-Describe fetches the path of mutations associated with each indicated sample's placement on the tree and prints the results.
+This command gets basic statistics about the input MAT.
 
 ### Specific Options
- 
-**-m**: File containing sample names for which mutation paths should be displayed (REQUIRED)
 
-## Filter
+**-s**: Write a tsv listing all samples in the tree and their parsimony scores (terminal branch length).
+
+**-c**: Write a tsv listing all clades in the tree and their occurrence over nodes in the tree.
 
-Filter removes an indicated set of samples from the input MAT, returning a smaller MAT.
+**-m**: Write a tsv listing all mutations in the tree and their occurrence count.
+
+## Annotate
+
+The annotate command takes a MAT protobuf file and a two-column tab-separated text file indicating sample names and lineage assignments. The software will automatically identify the best clade root for that lineage and save the assignment to each sample indicated, returning a new protobuf with these values stored.
```

Comment:
I would add, optionally, it takes a tsv file mapping clades to their respective internal node identifiers as input using `-C` and directly annotates clades without need for inference.

Reply:
oh, right. forgot you'd added that. Good stuff, will update